versions.txt

```diff
@@ -1,5 +1,5 @@
-behat/behat               3.19.0             Scenario-oriented BDD framework for PHP
+behat/behat               3.24.1             Scenario-oriented BDD framework for PHP
```

Comment:
Not exactly the focus of this PR, but does it make sense to have a versions.txt file with no composer.lock?